UnitsNet/CustomCode/Quantities/Area.extra.cs

```diff
@@ -41,5 +41,11 @@ public static Area FromCircleRadius(Length radius)
         {
             return VolumeFlow.FromCubicMetersPerSecond(area.SquareMeters * speed.MetersPerSecond);
         }
+
+        /// <summary>Get <see cref="LinearDensity"/> from <see cref="Area"/> times <see cref="Density"/>.</summary>
+        public static LinearDensity operator *(Area area, Density density)
+        {
+            return LinearDensity.FromGramsPerCentimeter(area.SquareCentimeters * density.GramsPerCubicCentimeter);
```

Comment:
Is there a particular reason you chose centimeters and grams in the implementation?
It works, but we try to stick with base units for the most part. 

Would you mind changing this to meters and kilograms? Same for the other implementations.

No need to change the tests, it's actually good to test with different units than the implementation.

Reply:
Makes sense.  Was just working in grams and centimeters and didn't think about it.  Changed now.